src/components/core/PivotTable.tsx

```diff
@@ -251,21 +254,23 @@ export const getGridDataSource = (
     onSuccess: (execution: Execution.IExecutionResponses, columnDefs: IGridHeader[]) => void,
     getGridApi: () => any,
     intl: InjectedIntl,
-    columnDefOptions: IColumnDefOptions = {}
+    columnDefOptions: IColumnDefOptions = {},
+    columnTotals?: AFM.ITotalItem[]
 ): IDatasource => ({
     getRows: ({ startRow, endRow, successCallback, sortModel }: IGetRowsParams) => {
         const execution = getExecution();
+
+        const resultSpecFinal: AFM.IResultSpec = resultSpec;
```

Comment:
{...resultSpec}

Reply:
https://petapixel.com/2015/07/22/this-is-how-everyone-names-their-final-psd-files/

Reply:
Updated